### PR TITLE
FIX cpr documentation

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

**Comment by AlvaroVega** [comment]
LGTM

File: doc/manuals/user/context_providers.md [hunk 1]
```diff
@@ -205,3 +205,6 @@ Some additional comments:
     If so, the registration fails and an error is returned.
     The OMA specification allows for regular expressions in entity id in registrations but as of now,
     the Context Broker doesn't support this feature.
+-   You should include entity type in the query/update in order ContextBroker being able to
```
Comment (kzangeli) [inline doc/manuals/user/context_providers.md]: in order **for the** ContextBroker **to be** able ...

File: doc/manuals/user/context_providers.md [hunk 2]
```diff
@@ -205,3 +205,6 @@ Some additional comments:
...
+    forward to Context Providers. Otherwise you could have problems as the one described in this
```
Comment (kzangeli) [inline doc/manuals/user/context_providers.md]: Otherwise you **may encounter** problems, **like** the one described ...

**Comment by fgalan** [comment]
PR was already merged. However, I'll do a new one with the typo-fixing raised by @kzangeli

**Pull request merged**